### ISSUE TITLE
request - Ministep add buttons for easier mappings

### ISSUE THREAD

**Issue by tastez** [mgunyho/PdArray#21]
Hi - just returning to the great Array module and since it is controlled by the Ministep, I am requesting some more features.

Ministep Reset button - this way its easy to map to something like Mindmeld Patchmaster and very quick to setup as a recorder of sequences with the Array module.

Increase, Decrease buttons could be cool to manually step through with Patchmaster controllers

**Comment by mgunyho** [comment]
Hi!

Thanks for the feature suggestion. Adding buttons for increment/decrement/reset could work, if the module was made a bit wider. However, I don't think I have much time to implement this in the near future. For now, you could use something like the VCV built-in Pulses module as the buttons, you could map those, right?

**Comment by tastez** [comment]
yeah there are definitely ways of doing it now with some cables and mappings so its nothing major! I would add just a tiny button so its easily mappable to midi or other control modules, not necessarily a big button for mouse clicks
